### ISSUE TITLE
@ character throws a SyntaxError "Invalid Character" during docker-compose build -t pleroma

### ISSUE THREAD

**Issue by maholloway** [Angristan/dockerfiles#2]
Steps to Reproduce:

Clone the repo
Follow the install steps, replacing host, secret_key_base, and email with valid values (pleroma.foo.bar, 8SwN5vykpfN6Xs2wGYztg87TdFgyhA/cMhnd7liNI3nWQI5fRLVcSByK92+MQw2m, admin@foo.bar)
Run docker-compose build -t pleroma

Console Output is:

Step 10/13 : RUN mix local.rebar --force     && mix local.hex --force     && mix deps.get     && mix compile
 ---> Running in 03cd8bb3a8c5                        
** (Mix.Config.LoadError) could not load config config/prod.secret.exs                                    
    ** (SyntaxError) config/prod.secret.exs:9: invalid character "@" (codepoint U+0040) in identifier: admin@foo
    (elixir) lib/code.ex:192: Code.eval_string/3     
    (mix) lib/mix/config.ex:188: Mix.Config.read!/2  
    (mix) lib/mix/config.ex:233: anonymous fn/3 in Mix.Config.read_wildcard!/2                            
    (elixir) lib/enum.ex:1899: Enum."-reduce/3-lists^foldl/2-0-"/3                                        
    (stdlib) erl_eval.erl:670: :erl_eval.do_apply/6  
    (stdlib) erl_eval.erl:878: :erl_eval.expr_list/6 
The command '/bin/sh -c mix local.rebar --force     && mix local.hex --force     && mix deps.get     && mix compile' returned a non-zero code: 1

Of note: entrypoint.sh is not present in the repo at the time of the submission of this issue.

**Comment by angristan** [comment]
No issue here, can you post the entire `secret.exs`?

**Comment by maholloway** [comment]
Must have been some unintended extra data somewhere on my end when I copied the config files.

I had rm -r 'd the directory so I started from scratch with dummy values (admin@my.domain) and the build seems to be puttering along nicely now.

tl;dr Starting over fixed it, issue was on my end!

**Comment by angristan** [comment]
Cool 👍
